### DISCUSSION TITLE
Welcome to GroupSpeedDial Discussions!

### DISCUSSION THREAD

**Discussion by fastaddons** [fastaddons/GroupSpeedDial#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Juraj-Masiar** [comment]
Hello everyone.
My name is Juraj and sometimes I will reply to you from my personal Github account. But it's still me! :)

**Comment by dagrace2** [comment]
Just trying to set this up. The "thumbnails" are weird. They look like just random pieces of the correct homepage or something. I googled a bit and found this:

Known issues:
- the default "Refresh dial" may not work for some highly protected pages like Facebook or banks - use "Refresh by visiting page" instead
- thumbnail creation is not perfect - some pages are loading their content asynchronously and thumbnail may be created incomplete - use delay function "+ 5s"

I can't find where that setting is. Looked a lot through all the apparent choices.

Is there another solution?

Thanks.

**Reply by dagrace2** [reply]
Oops. I'm unclear the relationship of github to GroupSpeedDail. I'm trying to install that add-on to my browser.

**Reply by Juraj-Masiar** [reply]
Two things comes to mind - if you have ultra wide monitor, the thumbnails may be too narrow.
There maybe also a mirror effect below the thumbnail if you resize your window (if it's not maximized). The reflection can be disabled in the Options page / Dials / "No image reflection".

But maybe something is wrong. What browser and OS are you using? And do you have some DPI scaling enabled on your monitor?

**Reply by N4RQ** [reply]
So, I use the option "Visual > Position" in the "Edit dial." Plenty of options for how the thumbnail appears. Could an option be included to set one of the thumbnail positions as the default one when a tab is created?
